src/.nuget/NuGet.Config

```diff
@@ -8,4 +8,8 @@
   <config>
     <add key="repositoryPath" value="../../packages" />
   </config>
+  <packageSources>
```

Comment:
This is the main change - add the roslyn-nightly feed from MyGet as a package source so that we can pick up a 1.2 version of the roslyn packages.